src/vue/backend/src/router/index.js

```diff
@@ -65,17 +72,26 @@ const routes = [
                     {
                         path: 'agents',
                         name: 'IMPress Agents Settings',
-                        component: () => import('@/views/settings/Agents')
+                        component: () => import('@/views/settings/Agents'),
+                        meta: {
+                            requires: [AGENTS]
```

Comment:
If agents etc is required to view the page, where would they go to activate it?

Reply:
@sgreenl5 http://jira.hq.idx.org/browse/WP-1299

Reply:
Any route will be accessible unless meta.strict and meta.requires are both present.
Basically, meta.strict = redirect, else show the view and disable related forms.